core/wasm/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "iroha_core_wasm"
```

Comment:
Not exactly a great name. It tells us nothing about what it does, other than that it's related to WASM and located in core (which we already knew).